examples/psp22/lib.rs

```diff
@@ -183,8 +199,7 @@ pub mod my_psp22 {
             let result = {
                 let _msg = build_message::<ContractRef>(address.clone())
                     .call(|contract| contract.transfer(address_of!(bob), 10, vec![]));
-                client.call_dry_run(&ink_e2e::alice(), &_msg, 0, None)
-                    .await
+                client.call_dry_run(&ink_e2e::alice(), &_msg, 0, None).await
             };
 
             assert!(matches!(result.return_value(), Err(PSP22Error::Custom(_))));
```

Comment:
add test for calling the `get_hated_account` function

Reply:
added more tests in #66